docs/reports/publish-a-report.md

```diff
@@ -0,0 +1,47 @@
+The report you've created so far is only a draft.
+Edit your report and complete the steps below to make it available for public viewing.
+
+## Switch to a published outputs
+Published reports require the output they're based on to also be published.
+
+Follow [the instructions on publishing your output](../jobs-site.md#publishing-outputs).
+
+Select a set of published outputs from the list:
+![List of published outputs](./images/job-server-published-outputs.jpg)
+
+!!! note
+    Published outputs can be in a draft status while they are being reviewed.
+    A badge saying "published" will be displayed next to those which have moved out of that state.
+
+Select the file you want to use for your report from the list on the left, and copy the direct URL which shows at the top of the file viewer.
+
+![Where to get the direct URL for a file](./images/job-server-direct-output-file-link-published.jpg)
+
+Replace the `Job server url` you currently have with this URL.you will need in the reports admin.
+
+
+## Create a DOI
```

Comment:
Is this something users should be doing or is it Bennett folk?

The reason I ask is we sign up to some principles/rules when making DOIs.... Jess (i think) will have more information if needed